src/mds/Server.cc

```diff
@@ -349,10 +349,16 @@ void Server::handle_client_session(MClientSession *m)
 	m->put();
 	return;
       }
+      //	This looks as if we are getting a seq that is higher than
+      //	expected.  I didn't notice any "wrap" handling, but the seqn is
+      //	64-bits ...
+      //
       if (m->get_seq() != session->get_push_seq()) {
 	dout(0) << "old push seq " << m->get_seq() << " != " << session->get_push_seq() 
-		<< ", BUGGY!" << dendl;
-	assert(0);
+ 		<< ", BUGGY!" << dendl;
+	mds->clog->warn() << "incorrect push seq " << m->get_seq() << " != " << session->get_push_seq() << ", dropping";
```

Comment:
Sure - I'll have to correct my emacs :-).  Is the formatting, etc the check that is failing?